yellowbrick/features/pcoords.py

```diff
@@ -328,51 +392,100 @@ def draw(self, X, y, **kwargs):
             Pass generic arguments to the drawing method
 
         """
+        if self.fast:
+            return self.draw_classes(X, y, **kwargs)
+        return self.draw_instances(X, y, **kwargs)
 
-        # Get shape of the sampled data
-        nrows, ncols = X.shape
+    def draw_instances(self, X, y, **kwargs):
+        """
+        Draw the instances colored by the target y such that each line is a
+        single instance. This is the "slow" mode of drawing, since each
+        instance has to be drawn individually. However, in so doing, the
+        density of instances in braids is more apparent since lines have an
+        independent alpha that is compounded in the figure.
```

Comment:
Is there a way to numerically represent braid density (a la silhouettes) that could be computed and added to the figure?

Reply:
Great idea, referred to in #452